cpp/src/arrow/adapters/orc/adapter.cc

```diff
@@ -44,20 +45,13 @@
 #include "arrow/util/macros.h"
 #include "arrow/util/range.h"
 #include "arrow/util/visibility.h"
-
 #include "orc/Exceptions.hh"
-#include "orc/OrcFile.hh"
+
+constexpr uint64_t kOrcWriterBatchSize = 100000;
```

Comment:
nit: power of 2 probably makes sense.  comment on how this was chosen (or if it was abitrary.  Also generally we avoid unsigned integers so if unsigned is required for API pursposes comment on it.

Reply:
Thanks! Yes these two constants have to be uint64_t due to the ORC APIs.